src/shared/components/palette/number/write-number-field.html

```diff
@@ -1,11 +1,12 @@
 <div class="form-group" [ngClass]="{'form-group-error': !numberControl.valid && numberControl.touched}">
 
   <label [for]="id()">
-    <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
-    <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
-    <span class="error-message" *ngIf="numberControl.errors && numberControl.touched">{{numberControl.errors | ccdFirstError}}</span>
+    <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>    
   </label>
+  <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
+  <span class="error-message" *ngIf="numberControl.errors && numberControl.touched">{{numberControl.errors | ccdFirstError:caseField.label}}</span>
 
-  <input class="form-control bottom-30" [id]="id()" type="number" [formControl]="numberControl">
+  <input class="form-control bottom-30" [ngClass]="{'govuk-input--error': numberControl.errors && numberControl.touched}"
+  [ngStyle]="{'border-width':numberControl.errors && numberControl.touched ? '2px':null}" [id]="id()" type="number" [formControl]="numberControl">
```

Comment:
Same as above

Reply:
As discussed removed the [ngStyle] attribute